controllers/controller.errors.js

```diff
@@ -0,0 +1,14 @@
+exports.handle404Errors = (req, res, next) => {
+    res.status(404).send({message: 'SORRY NO SUCH PATH ;('});
+};
+
+exports.handleOtherErrors = (err, req, res, next) => {
```

Comment:
Please remove the handleOtherError section as this is not required for the endpoints written so far.